UHERO/udaman
# Sorting select boxes and removing dbedt results

Testing steps
---

Test by going to the edit pages for categories, data_lists, measurements, and series. Check each select box. Check the index pages for the data portal objects.
